lantern_detection:
  min_distance: 30.0
  min_detections: 20
  min_yellow_points: 200
